templates/detailItems/adr.html

```diff
@@ -2,10 +2,6 @@
 	<option ng-repeat="option in ctrl.availableOptions" value="{{option.id}}">{{option.name}}</option>
 </select>
 <div>
-	<label for="details-pobox">{{ctrl.t.poBox}}</label>
```

Comment:
there is even one more we need to add - the second ' the extended address (e.g., apartment or suite number);'